apps/atomy-q/API/app/Services/QuoteIntake/DecisionTrailRecorder.php

```
@@ -56,28 +93,39 @@ private function record(
+            DecisionTrailEntry::query()->create([
+                'tenant_id' => $tenantId,
+                'comparison_run_id' => $comparisonRunId,
+                'rfq_id' => $rfqId,
+                'sequence' => $sequence,
+                'event_type' => $eventType,
+                'payload_hash' => $payloadHash,
+                'previous_hash' => $previousHash,
+                'entry_hash' => $entryHash,
+                'occurred_at' => now(),
+            ]);
+        });
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'record'. (undefined)

(StaticAccess)